### ISSUE TITLE
Can't start headless server due to UnsatisfiedLinkError exception in telemetry subsystem

### ISSUE THREAD

**Issue by gianluca-nitti** [MovingBlocks/Terasology#3021]
### What you were trying to do
Running an headless server, from the current version of the `develop` branch (c7d50f045620d5f10f90f704250571d01f486eec). I tried both `gradlew server` and the *Terasology PC (Headless)* IntelliJ run configuration.

### What actually happened
During the server initialization I get this exception:
```
java.lang.UnsatisfiedLinkError: no lwjgl64 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1864)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at org.lwjgl.Sys$1.run(Sys.java:72)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.lwjgl.Sys.doLoadLibrary(Sys.java:66)
	at org.lwjgl.Sys.loadLibrary(Sys.java:87)
	at org.lwjgl.Sys.<clinit>(Sys.java:117)
	at org.lwjgl.opengl.GLContext.<clinit>(GLContext.java:112)
	at org.lwjgl.opengl.GL11.glGetString(GL11.java:1792)
	at org.terasology.telemetry.metrics.SystemContextMetric.<init>(SystemContextMetric.java:82)
	at org.terasology.telemetry.Metrics.initialise(Metrics.java:46)
	at org.terasology.engine.subsystem.common.TelemetrySubSystem.postInitialise(TelemetrySubSystem.java:68)
	at org.terasology.engine.TerasologyEngine.postInitSubsystems(TerasologyEngine.java:267)
	at org.terasology.engine.TerasologyEngine.initialize(TerasologyEngine.java:199)
	at org.terasology.engine.TerasologyEngine.run(TerasologyEngine.java:368)
	at org.terasology.engine.Terasology.main(Terasology.java:142)
```

The game client with graphics (`gradlew game` or the *Terasology PC* IntelliJ run configuration) work correctly.

### How to reproduce
Just run `gradlew server`, or alternatively in IntelliJ select the *Terasology PC (Headless)* run configuration and click run. After a while the exception is thrown and the initialization hangs (no game is loaded).

### Log details and game version
Full output here: https://pastebin.com/RKxNEdVn
Game version: latest revision of the `develop` branch, commit hash c7d50f045620d5f10f90f704250571d01f486eec.

### Computer details
Probably not relevant, but anyway: Intel Core i7-4720HQ @ 2.60GHz, 16GB RAM, GTX 960M 4GB (with Nvidia Optimus, tried both the integrated and the dedicated GPU with the same result), Windows 10 Home.

**Comment by Cervator** [comment]
Okay I dug around a little and can confirm this issue.

When the telemetry system initializes it tries to record some OpenGL details in `SystemContextMetric.java` - however, when the game runs headless there is no `LwjglGraphics` initialized. See `Terasology.populateSubsystems()`

I don't remember if we have an easy way to determine if the current game process is running headless or not. The `Terasology` class is in the PC facade and is what initializes the engine, so the engine cannot at present access that class directly (there is a private static `isHeadless` boolean there)

In theory adding a condition that figures out the headless state to the few metrics in `SystemContextMetric` should resolve the issue.

**Comment by Cervator** [comment]
Closing as fixed by #3024!
